### DISCUSSION TITLE
GitHub contributions not showing accurate data

### DISCUSSION THREAD

**Discussion by hassancodess** [community/community#28804] [Other Feature Feedback, Questions, & Ideas]
I just installed Windows 11 on my Laptop. Configured the same username and password and whenever I make a commit it doesn't show on my Contribution graph!

So I use GitHub often and every day I make 10+ commits in different repositories but still it shows me 1-2 contributions

Does anyone know why is this happening?

**Comment by airtower-luna** [comment]
Username and password have nothing to do with getting commits connected to your account. Commits are recognized as yours if the author email recorded in the commit matches any of the email addresses on your account. So you'll need to [set the `user.email` configuration value](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address) so Git uses an address GitHub will recognize.
